wrapper/src/main/java/software/amazon/jdbc/util/SubscribedMethodHelper.java

```diff
@@ -48,4 +48,9 @@ public class SubscribedMethodHelper {
       "CallableStatement.executeUpdate",
       "CallableStatement.executeLargeUpdate"
   );
+
+  public static final List<String> XA_PREPARE_TRANSACTION = Arrays.asList(
+      "MysqlXAConnection.prepare",
```

Comment:
Where are we using this method names? In what wrapper class?

Reply:
I'm afraid a proper implementation of such feature should include a complete support of XAResource/XAConnection interfaces